src/webgpu/api/validation/createSampler.spec.ts

```diff
@@ -0,0 +1,19 @@
+export const description = `
+createSampler validation tests.
+
+Test Plan: 
+
+* lodMinClamp:
+  * < 0
```

Comment:
Should test several valid values, too, and large values. So, e.g. `-1`, `-Number.MIN_VALUE`, `0`, `0.5`, `1`, `10`, `Number.MAX_VALUE`. (Not 100% sure the CTS will be happy with all these values, let me know if there are problems.)

Reply:
It shouldn't be passing the test for -Number.MIN_VALUE right? could be int overflow, but createsampler is passing on  
-Number.MIN_VALUE right now

Reply:
oops, in the spec lodMinClamp is `float` not `double` so both Number.MIN_VALUE and Number.MAX_VALUE are non-representable values. I think MIN_VALUE will round to 0 and MAX_VALUE will become Infinity or NaN which is an IDL error?

Reply:
You can just come up with some smallish and largeish numbers instead like `1e-30` and `1e30`